cpp/benchmarks/io/parquet/parquet_reader_input.cpp

```diff
@@ -182,8 +182,9 @@ void BM_parquet_read_chunks(nvbench::state& state, nvbench::type_list<nvbench::e
   auto const d_type      = get_type_or_group(static_cast<int32_t>(DataType));
   auto const cardinality = static_cast<cudf::size_type>(state.get_int64("cardinality"));
   auto const run_length  = static_cast<cudf::size_type>(state.get_int64("run_length"));
-  auto const byte_limit  = static_cast<cudf::size_type>(state.get_int64("byte_limit"));
+  auto const chunk_read_limit  = static_cast<cudf::size_type>(state.get_int64("byte_limit"));
   auto const source_type = retrieve_io_type_enum(state.get_string("io_type"));
+  auto const data_size = static_cast<size_t>(state.get_int64("data_size"));
```

Comment:
`data_size` was hard-coded for all benchmarks. If you want to customize data size then let's do it consistently.